File: solutions/Virtual-Assistant/src/typescript/bot-solution/src/responses/card.ts
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

import { ICardData } from './cardDataBase';

export class Card {

    constructor (name: string, data : ICardData) {
        this.name = name;
        this.data = data;
    }

    public name: string = '';

    public data : ICardData = '';
}

File: sdk/csharp/libraries/microsoft.bot.solutions/Responses/Card.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using System;

namespace Microsoft.Bot.Solutions.Responses
{
    public class Card
    {
        public Card()
        {
        }

        public Card(string name)
        {
            Name = name;
        }

        public Card(string name, ICardData data)
        {
            Name = name;
            Data = data;
        }

        public string Name { get; set; }

        public ICardData Data { get; set; }
    }
}